pkg/keepalive/options.go

```diff
@@ -15,25 +15,44 @@
 package keepalive
 
 import (
+	"math"
 	"time"
 
 	"github.com/spf13/cobra"
 )
 
+const (
+	// Infinity is the maximum possible duration for keepalive values
+	Infinity = time.Duration(math.MaxInt64)
+)
+
 // Options defines the set of options used for grpc keepalive.
+// The Time and Timeout options are used for both client and server connections,
+// whereas MaxServerConnectionAge* options are applicable on the server side only
+// (as implied by the options' name...)
 type Options struct {
 	// After a duration of this time if the server/client doesn't see any activity it pings the peer to see if the transport is still alive.
 	Time time.Duration
 	// After having pinged for keepalive check, the server waits for a duration of Timeout and if no activity is seen even after that
 	// the connection is closed.
 	Timeout time.Duration
+	// MaxServerConnectionAge is a duration for the maximum amount of time a
+	// connection may exist before it will be closed by the server sending a GoAway.
+	// A random jitter is added to spread out connection storms.
+	// See https://github.com/grpc/grpc-go/blob/bd0b3b2aa2a9c87b323ee812359b0e9cda680dad/keepalive/keepalive.go#L49
+	MaxServerConnectionAge time.Duration // default value is infinity
```

Comment:
nit: consider moving the "default value..." comment into the comment above the parameter. Not sure how clever golang tooling is WRT documentation comments.

Reply:
Confirmed that the entire documentation is shown by `godoc`, including default value comment.